src/System.Net.Requests/tests/HttpWebRequestTest.netstandard1.7.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Net.Http;
+using System.Runtime.Serialization.Formatters.Binary;
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace System.Net.Tests
+{
+    public partial class HttpWebRequestTest17
```

Comment:
Class name must be the same with the file name (i.e. HttpWebRequestTest in this case).